THIRD_PARTY_LICENSES.txt

```diff
@@ -1793,7 +1793,7 @@ gomodules.xyz/jsonpatch/v2
 SPDX:Apache-2.0
 
 == Copyright
-(no copyright notices found)
+Copyright {yyyy} {name of copyright owner}
```

Comment:
Is this better than no copyright notice found?

Reply:
The copyright is copied from https://github.com/gomodules/jsonpatch/blob/release-2.0/LICENSE#L189.